### PR TITLE
#122 Get `flock` bot to 100% test coverage

### PR DESCRIPTION
**Pull request by novokrest**
Related to #122.
Also made some refactoring of bot's implementation.

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/flock/flock.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def get_flock_response(content: str, config: Dict[str, str]) -> str:
         return error
 
     if len(str(recipient_id)) > 30:
-        return recipient_id
+        return "Found user is invalid."
```
Comment (showell) [inline zulip_bots/zulip_bots/bots/flock/flock.py]: We should change the commit message here to reflect that you fixed the code itself.  I think `flock bot: Fix error message for invalid user.` would work.

**Comment by showell** [comment]
@novokrest Thanks for the code cleanup and tests here!  It appears this was one our "rougher around the edges" bots. 😄 

I think it might be worthwhile to take a pass at some of the longer commit messages, just to make their title lines a little shorter.  (It's not that big a deal, but it's a good habit to work on commit message hygiene, and people do appreciate it down the road.)  I think you can replace "local variable" with just "variable" or "local var" in some places.

I also added one inline comment where it appears you didn't just add tests but you also fixed the code.

File: zulip_bots/zulip_bots/bots/flock/flock.py (hunk 1 above)
Comment (novokrest) [reply zulip_bots/zulip_bots/bots/flock/flock.py]: @showell I updated commit message. Also I replaced `local variable` with `local var` in commit messages.

File: zulip_bots/zulip_bots/bots/flock/flock.py [hunk 2]
```diff
@@ -19,10 +19,8 @@ def find_recipient(res: str, to: str) -> str:
             return obj['id']
 
 # Returns User's ID, if not found, returns error message.
-def get_recipient_id(content: str, config: Dict[str, str]) -> str:
+def get_recipient_id(to: str, config: Dict[str, str]) -> str:
```
Comment (aero31aero) [inline zulip_bots/zulip_bots/bots/flock/flock.py]: In this file, why have we renamed `content` to `to` in one place and `recipient_name` in another?

File: zulip_bots/zulip_bots/bots/flock/flock.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ def find_recipient(res: str, to: str) -> str:
             return obj['id']
 
 # Returns User's ID, if not found, returns error message.
-def get_recipient_id(to: str, config: Dict[str, str]) -> str:
+def get_recipient_id(recipient_name: str, config: Dict[str, str]) -> str:
```
Comment (aero31aero) [inline zulip_bots/zulip_bots/bots/flock/flock.py]: I see you reversed it here, but this seems like these two commits could be squashed together.

File: zulip_bots/zulip_bots/bots/flock/flock.py [hunk 4]
```diff
@@ -18,6 +18,19 @@ def find_recipient_id(res: str, recipient_name: str) -> str:
         if recipient_name == obj['firstName']:
             return obj['id']
 
+# Male request to given flock URL and return two-element tuple
```
Comment (aero31aero) [inline zulip_bots/zulip_bots/bots/flock/flock.py]: Typo: `s/Male/Make`. Also, return *a* two-element tuple 😄  .

File: zulip_bots/zulip_bots/bots/flock/flock.py [hunk 5]
```diff
@@ -39,11 +39,11 @@ def get_recipient_id(recipient_name: str, config: Dict[str, str]) -> Tuple[Optio
     payload = {
         'token': token
     }
-    res, error = make_flock_request(USERS_LIST_URL, payload)
-    if res is None:
+    users, error = make_flock_request(USERS_LIST_URL, payload)
```
Comment (aero31aero) [inline zulip_bots/zulip_bots/bots/flock/flock.py]: I suppose the commits renaming res to users could be squashed into one.

File: zulip_bots/zulip_bots/bots/flock/fixtures/test_no_recipient_found.json [hunk 6]
```diff
@@ -6,9 +6,12 @@
             "token": "12345"
         }
     },
-    "response": {
-        "error": "No user found. Make sure you typed it correctly."
-    },
+    "response": [
+        {
+            "id": 1,
```
Comment (aero31aero) [inline zulip_bots/zulip_bots/bots/flock/fixtures/test_no_recipient_found.json]: There's a typo in the commit message: 

`flock: Refactor test_no_recipient_found test to incrase coverage.` -> incrase to increase.

Comment (novokrest) [reply zulip_bots/zulip_bots/bots/flock/fixtures/test_no_recipient_found.json]: @aero31aero Fixed

File: zulip_bots/zulip_bots/bots/flock/flock.py (hunk 5 above)
Comment (novokrest) [reply zulip_bots/zulip_bots/bots/flock/flock.py]: @aero31aero Squashed these commits into one.

File: zulip_bots/zulip_bots/bots/flock/flock.py (hunk 4 above)
Comment (novokrest) [reply zulip_bots/zulip_bots/bots/flock/flock.py]: @aero31aero Fixed typo in related commit.

**Comment by showell** [comment]
Merged, thanks @novokrest!

**Pull request merged**